src/main/java/org/prebid/server/util/BidderUtil.java

```diff
@@ -18,6 +18,10 @@ public static boolean isValidPrice(Price price) {
         return isValidPrice(price.getValue()) && StringUtils.isNotBlank(price.getCurrency());
     }
 
+    public static boolean isNotEqualsIgnoreCase(String impBidFloorCur, String bidderCurrency) {
+        return !StringUtils.equalsIgnoreCase(impBidFloorCur, bidderCurrency);
+    }
+
```

Comment:
Pls, remove this method. It's purpose it not clear from it's name. I guess it's better to inline calls like that.